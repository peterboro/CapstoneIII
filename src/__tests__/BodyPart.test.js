import React from 'react';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import BodyPart from '../components/BodyPart';

const TestRenderer = require('react-test-renderer');

it('Renders body part correctly', () => {
  const tree = TestRenderer
    .create(
      <Provider store={store}>
        <BodyPart />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});