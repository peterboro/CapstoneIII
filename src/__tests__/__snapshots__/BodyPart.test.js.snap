// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders body part correctly 1`] = `
<div
  className="bodyPart-card css-1gwnsqj-MuiStack-root"
  onClick={[Function]}
  type="button"
>
  <img
    alt="dumbbell"
    src={
      Object {
        "$$typeof": Symbol(react.memo),
        "compare": null,
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        },
      }
    }
    style={
      Object {
        "height": "40px",
        "width": "40px",
      }
    }
  />
  <p
    className="MuiTypography-root MuiTypography-body1 css-wtph2a-MuiTypography-root"
  >
     
  </p>
</div>
`;
